### DISCUSSION TITLE
should the energy in erergy.raw be the free energy F or the internal energy E=F+TS

### DISCUSSION THREAD

**Discussion by pig-pig-man2020** [deepmodeling/dpdata#309] [Q&A]
hello, i am a novice in molecular dynamics simulation and i have a practical problem.
In the case of high temperature (for example T=1eV), TS item in internal energy E=F+TS is not negligible for high precision. potential energy surface build by dp should be the free energy F or the internal energy E=F+TS ?
dpdata just extract free energy F as energy in energy.raw/.npy, is it suitable in high temperature situation ?

**Comment by J-You** [comment]
I am also looking for an answer to this question. Based on my current understanding, I believe that the ground state energy $E_0$ (in VASP)
should be fitted rather than the free energy, even for high-temperature systems.
